SPECS/freefont/freefont.spec

```diff
@@ -24,13 +25,19 @@ UCS (Universal Character Set).
 %install
 mkdir -p %{buildroot}%{_fontdir}
 mv *.ttf %{buildroot}%{_fontdir}/
+fc-cache
```

Comment:
What's the need to run `fc-cache` here? If it generates any files, which need to be installed with `freefont`, then keep in mind, that you will need to also move them to some subdirectory of `%{buildroot}` and then include them in the `%files` section. Without that all changes done by `fc-cache` will remain local to the package's build environment and will get discarded right after the package finished building.

Reply:
Oh, I was doing that because freefont/fontconfig instructions were to run fc-cache after installing, but makes sense that doesn't apply to us. I'll just remove it :)

Reply:
If that's the guidance from upstream, then we should run `fc-cache` in a `%post` section and add `Requires(post): fontconfig` to this package's requirements. That way, the cache is updated on users' machines post-install.